backend/app.py

```diff
@@ -5,6 +5,7 @@
 import sys
```

Comment:
All alternatives for a hierarchy should be a part of the get hierarchy response.

Reply:
I've added this functionality. There's something finicky that I talked about on discord and will share in an updated comment.